### ISSUE TITLE
Country dashboards for hosted portals

### ISSUE THREAD

**Issue by jocelynpender** [gbif/hosted-portals#99]
As discussed in the meeting on Mar. 4th, we are interested in having the country-level data summaries displayed on our hosted portal, as seen on [GBIF's summary page for Canada](https://www.gbif.org/country/CA/summary).

Summary metrics are helpful to orient users to the scoped data.
@timrobertson100 pointed out that there is a difference between summaries of present-day data and time-series data summaries. Summaries of time-series data are more difficult to display, requiring snapshots to be taken and held. Given that challenge, I don't think visualizing time series is a priority for us.
Simple things like **occurrences per dataset** are useful, showing how present-day data break down, and who is publishing data about Canada, etc. @jmacklin feel free to add here.

**Please close this issue if it is a duplicate. I couldn't find it.**

**Comment by MortenHofft** [comment]
Your hosted portal currently shows [data _about_ Canada](https://github.com/gbif/hp-canadian-biodiversity/commit/116a858f3e642e5aae2528d2bd821c2437ca2ab6)

So in your case a dashboard would correspond to the [about tab](https://www.gbif.org/country/CA/about) (image below). Is that correctly understood and what you would like to see? @jenningsdt you have mentioned the dashboard as well - does this capture what you imagine as well?

![Canada](https://user-images.githubusercontent.com/6759267/110303834-29383380-7ffb-11eb-9998-99956c73cefd.png)

**Comment by jenningsdt** [comment]
Yes, the About page on the country pages as shown above captures what we were thinking for a Regional dashboard. That would be a great start! Eventually, it would be great to have other elements like publications on a regional scale (like More... Publications About).

**Comment by jocelynpender** [comment]
Yes, this is what we're thinking as a great start! I agree with @jenningsdt it would be cool to add other elements, one day.

**Comment by aserawayneasera** [comment]
Yes, this is what we would like also if possible, since the Pacific hosted portal is showing data for a collective of 14 plus countries.

**Comment by giannijacot** [comment]
We would also be interested in showing predefined dashboards in our hosted portal.

I see that a commit has been made for this issue, is there already a way to do it ?

A way that would work for us is, for exemple, to configure dashboards in the config.js file that would show as default ones in the dashboard tab on the occurences page. Showing predefined dashboards on other pages of the hosted portal would also work for us.

**Comment by MortenHofft** [comment]
There is currently no option to predefine or lock the occurrence/search - dashboard.

But you can add custom predefined metrics to markdown pages. The demo site shows an example https://hp-theme.gbif-staging.org/layout/compose . It is one of those building blocks / predefined layouts that can be used. 
The data used for the block is here https://github.com/gbif/jekyll-hp-base-theme/blob/master/_data/compose/dashboard.yml

**Comment by giannijacot** [comment]
Thanks for your answer. Adding dashboards in markdown pages is a good solution for our needs.

**Comment by MortenHofft** [comment]
I'mc closing this due to lack of activity and because we have the generic dashboards that seem to solve most of not all. For any missing dashboard features lets look at adding those instead.
